packages/sdk-core/src/modules/database/model/BacktraceDatabaseRecord.ts

```diff
@@ -1,15 +1,34 @@
 import { BacktraceAttachment } from '../../../model/attachment';
 import { BacktraceData } from '../../../model/data/BacktraceData';
 
-export interface BacktraceDatabaseRecord {
+export interface ReportBacktraceDatabaseRecord {
+    readonly type: 'report';
     readonly data: BacktraceData;
     readonly id: string;
     readonly hash: string;
     readonly timestamp: number;
+    readonly sessionId?: string;
```

Comment:
how can we benefit from sessionId?

Reply:
We can lock specific sessions and not all of them. This was used before, but was hidden behind `application.session` attribute.